sahi/predict.py

```diff
@@ -315,6 +317,10 @@ def predict(
     postprocess_match_threshold: float = 0.5,
     postprocess_class_agnostic: bool = False,
     export_visual: bool = False,
+    input_video: bool = False,
```

Comment:
we should remove this parameter and check if source is video by looking its suffix (mp4, avi, mpeg etc)

Reply:
Good idea for video control.

Reply:
you can get the suffix as:
```python
from pathlib import Path
suffix = Path(file_path).suffix
```
source: https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.suffix